### PR TITLE
explain the usage of exp and iat jwt claims

### PR DESCRIPTION
**Pull request by Sakurann**
Issue #133.

partially addresses issue #39 though does not give clarity what is mandated, yet.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/151.html" title="Last updated on Sep 12, 2023, 5:22 PM UTC (1788441)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/151/9e717da...1788441.html" title="Last updated on Sep 12, 2023, 5:22 PM UTC (1788441)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -600,6 +600,13 @@ <h2>Protected Header Parameters</h2>
         <code>alg</code>, <code>iat</code>,
         <code>exp</code> and <code>cnf</code>.
       </p>
+      <p>
+        When <code>exp</code> or <code>exp</code> JWT claims are present,
```
Comment (David-Chadwick) [inline index.html:604]: there is a typo here. Replace first exp with iat

**Review by David-Chadwick — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Sakurann) [inline index.html:604]:
```suggestion
        When <code>iat</code> or <code>exp</code> JWT claims are present,
```

File: index.html [hunk 2]
```diff
@@ -600,6 +600,13 @@ <h2>Protected Header Parameters</h2>
...
+        they represent issuance and expiration time of a signature, respectively.
```
Comment (David-Chadwick) [inline index.html:605]: they represent the issuance and expiration times of the signature, respectively.

File: index.html [hunk 3]
```diff
@@ -600,6 +600,13 @@ <h2>Protected Header Parameters</h2>
...
+        Note that this is different from <code>validFrom</code> and <code>validUntil</code>
+        defined in <a data-cite="VC-DATA-MODEL#validity-period">Validity Period</a>
+        that represent validity of the data that is being secured.
```
Comment (David-Chadwick) [inline index.html:608]: that represents the validity...

File: index.html [hunk 4]
```diff
@@ -600,6 +600,13 @@ <h2>Protected Header Parameters</h2>
         <code>alg</code>, <code>iat</code>,
         <code>exp</code> and <code>cnf</code>.
       </p>
+      <p>
+        When <code>exp</code> or <code>exp</code> JWT claims are present,
+        they represent issuance and expiration time of a signature, respectively.
+        Note that this is different from <code>validFrom</code> and <code>validUntil</code>
```
Comment (David-Chadwick) [inline index.html:606]: Note that they are different from the <code>validFrom</code> and <code>validUntil</code> properties

**Review by David-Chadwick — CHANGES_REQUESTED** [review]

File: index.html [hunk 5]
```diff
@@ -600,6 +600,13 @@ <h2>Protected Header Parameters</h2>
         <code>alg</code>, <code>iat</code>,
         <code>exp</code> and <code>cnf</code>.
       </p>
+      <p>
+        When <code>iat</code> or <code>exp</code> JWT claims are present,
```
Comment (selfissued) [inline index.html:604]: When -> When the

**Review by selfissued — APPROVED** [review]
I approve, subject to the editorial suggestions being incorporated.

**Review by OR13 — APPROVED** [review]

File: index.html (hunk 5 above)
Comment (TallTed) [reply index.html:604]: @selfissued's suggestion plus another
```suggestion
        When the <code>iat</code> and/or <code>exp</code> JWT claims are present,
```

File: index.html [hunk 6]
```diff
@@ -600,6 +600,13 @@ <h2>Protected Header Parameters</h2>
...
+        they represent issuance and expiration time of a signature, respectively.
+        Note that this is different from <code>validFrom</code> and <code>validUntil</code>
```
Comment (TallTed) [inline index.html:606]:
```suggestion
        Note that these are different from the <code>validFrom</code> and <code>validUntil</code>
```

File: index.html [hunk 7]
```diff
@@ -600,6 +600,13 @@ <h2>Protected Header Parameters</h2>
...
+        defined in <a data-cite="VC-DATA-MODEL#validity-period">Validity Period</a>
+        that represent validity of the data that is being secured.
```
Comment (TallTed) [inline index.html:608]:
```suggestion
        that represent the validity of the data that is being secured.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Sakurann) [reply index.html:605]:
```suggestion
        they represent the issuance and expiration time of the signature, respectively.
```

File: index.html (hunk 4 above)
Comment (Sakurann) [reply index.html:606]:
```suggestion
        Note that they are different from <code>validFrom</code> and <code>validUntil</code> properties
```

Comment (Sakurann) [reply index.html:606]:
```suggestion
        Note that these are different from the <code>validFrom</code> and <code>validUntil</code> properties
```

File: index.html (hunk 3 above)
Comment (Sakurann) [reply index.html:608]:
```suggestion
        that represent the validity of the data that is being secured.
```

**Comment by Sakurann** [comment]
(requested re-review by @selfissued by mistake..)

**Review by selfissued — APPROVED** [review]

**Review by TallTed — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-09-14-vcwg#section6-3) on 2023-09-14

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 6.3. explain the usage of exp and iat jwt claims (pr vc-jose-cose#151)

_See github pull request [vc-jose-cose#151](https://github.com/w3c/vc-jose-cose/pull/151)._




**Kristina Yasuda:** please open issues before merging that PR.  

**Michael Jones:** this PR clarifies how iat/exp should be used. very small.  

**Manu Sporny:** +1 to this PR. makes good distinction between dates associated with proof and VCs. +1 to merge.  

**Brent Zundel:** no objections. open for more then a week - merge it.  

</details>

**Pull request merged**